lib/main_common.pm

```diff
@@ -596,33 +596,32 @@ sub load_iso_in_external_tests {
 }
 
 sub load_x11regression_documentation {
-    if (check_var("DESKTOP", "gnome")) {
-        loadtest "x11regressions/gnote/gnote_first_run";
-        loadtest "x11regressions/gnote/gnote_link_note";
-        loadtest "x11regressions/gnote/gnote_rename_title";
-        loadtest "x11regressions/gnote/gnote_undo_redo";
-        loadtest "x11regressions/gnote/gnote_edit_format";
-        loadtest "x11regressions/gnote/gnote_search_all";
-        loadtest "x11regressions/gnote/gnote_search_body";
-        loadtest "x11regressions/gnote/gnote_search_title";
-        loadtest "x11regressions/evince/evince_open";
-        loadtest "x11regressions/evince/evince_view";
-        loadtest "x11regressions/evince/evince_rotate_zoom";
-        loadtest "x11regressions/evince/evince_find";
-        loadtest "x11regressions/gedit/gedit_launch";
-        loadtest "x11regressions/gedit/gedit_save";
-        loadtest "x11regressions/gedit/gedit_about";
-        if (!is_tumbleweed) {
-            loadtest "x11regressions/libreoffice/libreoffice_mainmenu_favorites";
-            loadtest "x11regressions/evolution/evolution_prepare_servers";
-            loadtest "x11regressions/libreoffice/libreoffice_pyuno_bridge";
-        }
-        loadtest "x11regressions/libreoffice/libreoffice_mainmenu_components";
-        loadtest "x11regressions/libreoffice/libreoffice_recent_documents";
-        loadtest "x11regressions/libreoffice/libreoffice_default_theme";
-        loadtest "x11regressions/libreoffice/libreoffice_open_specified_file";
-        loadtest "x11regressions/libreoffice/libreoffice_double_click_file";
-    }
+    return unless check_var('DESKTOP', 'gnome');
+    loadtest "x11regressions/gnote/gnote_first_run";
+    loadtest "x11regressions/gnote/gnote_link_note";
+    loadtest "x11regressions/gnote/gnote_rename_title";
+    loadtest "x11regressions/gnote/gnote_undo_redo";
+    loadtest "x11regressions/gnote/gnote_edit_format";
+    loadtest "x11regressions/gnote/gnote_search_all";
+    loadtest "x11regressions/gnote/gnote_search_body";
+    loadtest "x11regressions/gnote/gnote_search_title";
+    loadtest "x11regressions/evince/evince_open";
+    loadtest "x11regressions/evince/evince_view";
+    loadtest "x11regressions/evince/evince_rotate_zoom";
+    loadtest "x11regressions/evince/evince_find";
+    loadtest "x11regressions/gedit/gedit_launch";
+    loadtest "x11regressions/gedit/gedit_save";
+    loadtest "x11regressions/gedit/gedit_about";
+    if (!is_tumbleweed) {
```

Comment:
could you replace that with whatever it *should* match? E.g. `!sle_version_at_least('15') && !leap_version_at_least('15')`?

Reply:
fixed